modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -298,7 +298,16 @@ ifdef::ibm-z,ibm-z-kvm[]
 <15> Add the `additionalTrustBundle` parameter and value. The value must be the contents of the certificate file that you used for your mirror registry. The certificate file can be an existing, trusted certificate authority or the self-signed certificate that you generated for the mirror registry.
 endif::ibm-z,ibm-z-kvm[]
 ifndef::openshift-origin[]
-<16> Provide the `imageContentSources` section from the output of the command to mirror the repository.
+<16> Provide the `imageContentSources` section according to the output of the command you used to mirror the repository. 
++
+[IMPORTANT]
+====
+Provide the input according to the command you used for mirroring:
+
+* When using `oc adm release mirror` command: use the `imageContentSources` section from the output of `oc adm release mirror` command
+* When using `oc-mirror` command: the `repositoryDigestMirrors` section of `ImageContentSourcePolicy`
```

Comment:
I wrote <insert_similar_text_here> because I'm not sure how to take this information. Can they even use the `oc mirror` command if ImageContentSourcePolicy is deprecated? Wondering why they would use this at all. If it can be removed, then I think the second two bullet points could go?

Reply:
@stevsmit even though ImageContentSourcePolicy is deprecated at 4.14, it still exists, is usable and supported. SAMSUNG is using it now, which might be why the devs have been so slow about actually REMOVING the yaml section.

Reply:
However, I did make your suggested changes, thank you!